app/src/main/java/me/ccrama/redditslide/Fragments/SubmissionsView.java

```diff
@@ -443,7 +443,8 @@ private void refresh() {
     }
 
     public void forceRefresh() {
-        rv.scrollToPosition(0);
+        toolbarScroll.toolbarShow();
```

Comment:
This was necessary because `scrollToPosition` calls a LayoutChange even, not a Scroll event, so we couldn't directly leverage the existing logic on the Helper.